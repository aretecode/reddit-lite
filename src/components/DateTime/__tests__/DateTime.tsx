import * as React from 'react'
import { render } from 'react-testing-library'
import { DateTime } from '../'
import * as exported from '../'

describe('DateTime', () => {
  it('should match export snapshot', () => {
    expect(exported).toMatchSnapshot()
  })
  it('should render without errors', () => {
    const { container } = render(<DateTime />)
    expect(container).toMatchSnapshot()
  })
})
