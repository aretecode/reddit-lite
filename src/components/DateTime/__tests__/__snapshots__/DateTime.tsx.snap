// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DateTime should match export snapshot 1`] = `
Object {
  "DateTime": [Function],
  "StyledTime": Object {
    "$$typeof": Symbol(react.forward_ref),
    "attrs": Array [],
    "componentStyle": ComponentStyle {
      "componentId": "sc-bdVaJa",
      "isStatic": true,
      "rules": Array [
        "
  display: inline-flex;
",
      ],
    },
    "displayName": "styled.time",
    "foldedComponentIds": Array [],
    "render": [Function],
    "styledComponentId": "sc-bdVaJa",
    "target": "time",
    "toString": [Function],
    "warnTooManyClasses": [Function],
    "withComponent": [Function],
  },
  "default": [Function],
}
`;

exports[`DateTime should render without errors 1`] = `
.c0 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
}

<div>
  <time
    class="c0"
  />
</div>
`;
